templates/base.html

```diff
@@ -8,8 +8,13 @@
 		<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap-theme.min.css">
 		<script src="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>
 		<link rel="stylesheet" href="{% static 'css/style.css' %}">
+		<link rel="stylesheet" href="{% static 'css/footer.css' %}">
```

Comment:
OK. Will move all to style.css.

On Mon, Jan 5, 2015 at 10:49 AM, Erika <notifications@github.com> wrote:

> In templates/base.html
> <https://github.com/CodeCatz/TrackCat/pull/47#discussion-diff-22453070>:
>
> > @@ -8,8 +8,13 @@
> >  		<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap-theme.min.css">
> >  		<script src="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>
> >  		<link rel="stylesheet" href="{% static 'css/style.css' %}">
> > +		<link rel="stylesheet" href="{% static 'css/footer.css' %}">
>
> @anuschka <https://github.com/anuschka> no need for additional .css file.
> Everything can be in one having multiple files is not optimal as the client
> makes request for everyone separately.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CodeCatz/TrackCat/pull/47/files#r22453070>.
>